src/main/java/github/weichware10/toolbox/Main.java

```diff
@@ -0,0 +1,71 @@
+package github.weichware10.toolbox;
+
+import javafx.application.Application;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.stage.WindowEvent;
+
+/**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.